docs/coding-style.md

```diff
@@ -87,6 +87,13 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
     someEnumerable.Where(x => x.Condition1
                               && x.Condition2);
     ```
+  * Exception from this rule: If the first argument is a lambda expression, the parameter name and the arrow token should go in the same line with the invocation.
```

Comment:
To keep it simple and single-line
```suggestion
  * Exception from this rule: Lambda parameter name and arrow token should be on the same line as the invocation.
```
It intentionally ommits the "first lambda argument" part. That one is visible from example and not doable anywhere else. I'd be fine with having it as 2nd argument too, actually.